src/topology/metric_space/basic.lean

```diff
@@ -863,6 +863,12 @@ lemma closed_ball_Icc {x r : ℝ} : closed_ball x r = Icc (x-r) (x+r) :=
 by ext y; rw [mem_closed_ball, dist_comm, real.dist_eq,
   abs_sub_le_iff, mem_Icc, ← sub_le_iff_le_add', sub_le]
 
+-- This lemma could go somewhere else, but I did not find a better fit for it.  Feel
+-- free to suggest where it may be added.  Here, all the needed imports are available!
```

Comment:
I think this belongs in `ordered_group.lean`. Importing `data.set.intervals.basic` there is unlikely to cause problems.

Reply:
Ok, I will wait to see if someone has strong opinion about where to move this file or about adding imports, and will go with the final suggestion.

Reply:
Sorry, I'm wrong - these need to go in `data.set.intervals.basic`, since that imports `ordered_group`